src/transforms/caching.js

```diff
@@ -33,6 +34,7 @@ var cacheExempt = [
   'poisson',
   'randomInteger',
   'tensorGaussian',
+  'tensorLaplacian',
```

Comment:
These need to match the name of the (auto-generated) sampling helpers, so `laplace` and `tensorLaplace`.